package.json

```diff
@@ -6,26 +6,31 @@
   "type": "module",
   "devDependencies": {
     "@playwright/test": "^1.35.1",
+    "@prisma/client": "4.15.0",
```

Comment:
This is a server side dependency so I'm surprised this works, but I guess the generated Prisma client doesn't have any requirements to the Prisma package itself? 🤔🤔🤔

Reply:
It sure seems that way, at least it seems to work OK. Prisma appears to copy the 15 MB native .so deps when it generates the client, so they're there twice (once in db/ and once in node_modules/prisma/engines).